index.js

```diff
@@ -90,9 +93,19 @@
           }
         }.bind(this), RATE_LIMIT);
       }.bind(this);
+
+      /* Adding listeners to page events */
       document.addEventListener("visibilitychange", this._rcv_fn);
       document.addEventListener("scroll", this._rcv_fn);
       window.addEventListener("resize", this._rcv_fn);
+
+      /* Adding scroll listeners to all element's parents */
+      element = domnode;
```

Comment:
do we need this assignment? Why not just use `domnode` in the following few lines?